### ISSUE TITLE
VMS: WP Sign-in Sheet Layout tweaks

### ISSUE THREAD

**Issue by WaBirder57** [washingtontrails/vms#369]
From VMS Crew Corner tab of a work party the Crew Leader can view and print the roster for the Work Party.  Can you make the following adjustments:

- Adjust the column widths for the write in portion of this sheet to have more room for the Name and Signature (say 20% wider), perhaps sacrifice Address column Width.  
- Make the font size for the Liability language at the top a little larger.  Think dark wet morning at the trail head where folks will be reading this and signing in perhaps with out reading glasses.  But then again it all needs to fit, perhaps 10pt or 12pt font size.

**Comment by davisagli** [comment]
You mean for the Sign-In Sheet, not the Contact List, right?

**Comment by WaBirder57** [comment]
Correct. Sign-in sheet.

**Comment by cewing** [comment]
This should now be corrected.  I've increased the font size for the waiver text while leaving the rest of the text on the page the same size.  I've set the total width of the name and signature columns in the unregistered sign up table to be a bit more than 1/3 of the total width of the table.  

I've also set a print rule that should ensure that for most browsers (Chrome being the only exception I know of) the headers for table columns will repeat on new pages if the table is forced to break across two pages.
